float32[3] origin
float32[3] head
float32[3] leftShoulder
float32[3] rightShoulder
float32[3] hipTorso
string timestamp
